web-client/src/translations/translations.json

```diff
@@ -2,12 +2,18 @@
   "en": {
     "common": {
       "welcome": "Welcome!",
+      "continue": "Continue",
       "login": {
         "title": "Reach4Help",
         "sub_title": "Please Note",
         "info": "We are only allowing Facebook Users with a verified account 6 months or older for safety. \nWe are working hard to bring more ways to login in the future. \nThank you for your patience.",
         "facebookButtonLabel": "Continue with Facebook"
       },
+      "phoneNumber": {
+        "title": "Welcome",
```

Comment:
The welcome translations already exist to both en/pt

Reply:
Already exist but have an exclamation mark after

Reply:
I can maybe change the the screen that is using that string and add the exclamation mark there

Reply:
Yes I know, but it was created just to bootstrap the translations, so you can change it